cryptofeed/backends/postgres.py

```diff
@@ -40,11 +40,12 @@ def __init__(self, host='127.0.0.1', user=None, pw=None, db=None, table=None, nu
         self.db = db
         self.pw = pw
         self.host = host
+        self.port = port
         self.max_batch = max_batch
 
     async def _connect(self):
         if self.conn is None:
-            self.conn = await asyncpg.connect(user=self.user, password=self.pw, database=self.db, host=self.host)
+            self.conn = await asyncpg.connect(user=self.user, password=self.pw, database=self.db, host=self.host, port=self.port)
```

Comment:
will this work if its none? Shouldnt the default be the default Postgres port, 5432?

Reply:
It will work if `None` will be passed to `asyncpg.connect` since this is also the default value, see also [their documentation](https://magicstack.github.io/asyncpg/current/api/index.html#connection). It will automatically fall back to port `5432` if no port is provided using the `port` argument or the `PGPORT` environment variable.